lib/query.js

```diff
@@ -313,6 +313,21 @@ export default class Query {
     return this;
   }
 
+  limitResults(num) {
+    this.limit = num;
+    return this;
+  }
+
+  skipResults(num) {
+    this.offset = num;
+    return this;
+  }
+
+  retrievePage(num) {
+    this.page = num;
+    return this;
+  }
```

Comment:
@ben181231 but the names are already used by the fields, if I override them it will break backwards compatibility....